core/vm/instructions.go

```diff
@@ -353,9 +353,10 @@ func opReturnDataCopy(pc *uint64, interpreter *EVMInterpreter, scope *ScopeConte
 
 func opExtCodeSize(pc *uint64, interpreter *EVMInterpreter, scope *ScopeContext) ([]byte, error) {
 	slot := scope.Stack.peek()
-	cs := uint64(interpreter.evm.StateDB.GetCodeSize(slot.Bytes20()))
+	address := slot.Bytes20()
+	cs := uint64(interpreter.evm.StateDB.GetCodeSize(address))
 	if interpreter.evm.chainRules.IsPrague {
-		statelessGas := interpreter.evm.Accesses.TouchAddressOnReadAndComputeGas(slot.Bytes(), uint256.Int{}, utils.CodeSizeLeafKey)
+		statelessGas := interpreter.evm.Accesses.TouchAddressOnReadAndComputeGas(address[:], uint256.Int{}, utils.CodeSizeLeafKey)
```

Comment:
The replay panicked just before here, but this also requires a fix.

Reply:
ah ok so you're not actually removing it. nvm, it'll be removed soon enough.